app/graphql/types/query_type.rb

```diff
@@ -33,7 +33,7 @@ def event(id:)
     end
 
     def event_connection(**args)
-      Event.all
+      Event.sort_by_time.all
```

Comment:
Does this filter from `Time.now`?

Reply:
This orders by `dtstart` descending.

Reply:
I think there's no point returning events that are in the past so maybe should update the filter? (assume you mean ascending too)

Reply:
as i understand- the events are returned in the same paginated way as partners, just ordered by dtstart so if you have an offset of 100 then all the events will be returned after 100 ordered by dtstart. there is currently no way to specify a 'from date' field in the query